pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/AggregateOperator.java

```diff
@@ -192,6 +194,8 @@ private TransferableBlock produceAggregatedBlock() {
       if (rows.isEmpty()) {
         return TransferableBlockUtils.getEndOfStreamTransferableBlock();
       } else {
+        ColumnDataType[] storedTypes = _resultSchema.getStoredColumnDataTypes();
+        rows.forEach(row -> TypeUtils.convertRow(row, storedTypes));
```

Comment:
It is not clear to me why we add the conversion here. TBH the problem may be that is not clear to me when a value uses the stored type or the execution type.

What happens if we have two aggregation stages? Something like:
```
select distinct(c) from (
  select A, count() as c from T
    group by A
  ) as TableExpr
```

Do we convert values twice?

Reply:
Actually I found that we do call `convertRow()` twice, once in executor and once here.. Removing it from here since it is more clear to convert in the executor